build-tools/packages/build-cli/docs/check.md

```diff
@@ -97,6 +98,32 @@ EXAMPLES
 
 _See code: [src/commands/check/changeset.ts](https://github.com/microsoft/FluidFramework/blob/main/build-tools/packages/build-cli/src/commands/check/changeset.ts)_
 
+## `flub check latestVersions VERSION PACKAGE_OR_RELEASE_GROUP`
+
+Determines if an input version matches a latest minor release version. Intended to be used in the Fluid Framework CI pipeline only.
```

Comment:
Can you add some example sceanrios to help clarify? For example, "If the inpout version is 2.1.3, and the most recent released version in the v2 major version series is 2.2.1, this command will fail with an error. If the input version is 2.1.3 and the most recent released version is 2.2.5, the command will succeed." Something like that.